From 44b8884d82fb888da72ea71dfa52958fc8b980a3 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Fri, 26 Aug 2016 04:37:24 +0200
Subject: [PATCH] libselinux: Fully libcrypto_static include in static library

 * Solves external/libselinux/./src/android.c:999:
    error: undefined reference to 'SHA1' for toybox_static

As well as the same error for static make ext4fs in later versions
Change-Id: I323d818283110b9ca1ea9cefaa72f4e5320e3a19
---
 Android.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 659232e..e25a499 100644
--- a/Android.mk
+++ b/Android.mk
@@ -45,9 +45,8 @@ include $(CLEAR_VARS)
 LOCAL_SRC_FILES := $(common_SRC_FILES) $(common_HOST_FILES) src/android.c
 LOCAL_MODULE:= libselinux
 LOCAL_MODULE_TAGS := eng
-LOCAL_STATIC_LIBRARIES := libcrypto_static
 LOCAL_C_INCLUDES := $(LOCAL_PATH)/include
-LOCAL_WHOLE_STATIC_LIBRARIES := libpcre libpackagelistparser
+LOCAL_WHOLE_STATIC_LIBRARIES := libcrypto_static libpcre libpackagelistparser
 # 1003 corresponds to auditd, from system/core/logd/event.logtags
 LOCAL_CFLAGS := -DAUDITD_LOG_TAG=1003
 # mapping.c has redundant check of array p_in->perms.
-- 
2.11.0

